### PR TITLE
fix: replace cron.s.#### to cron.s

### PR DESCRIPTION
**Pull request by kuyh**
http://git.sphd.io/support/stt-issues/issues/2264

### CODE REVIEW

**Comment by heycalmdown** [comment]
Could you please squash the commits into one when you merge this?

**Comment by kuyh** [comment]
ok I always use squash and merge on github haha

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "build": "gulp build",
     "watch": "gulp build && gulp watch",
     "clean": "gulp clean",
-    "test": "DEBUG=* ISLAND_LOGGER_LEVEL=crit gulp test",
+    "test": "DEBUG=* ISLAND_LOGGER_LEVEL=crit gulp",
```
Comment (Ohen) [inline package.json]: is gulp's default command to be `test`?

Comment (kuyh) [reply package.json]: Yes. default is test.

and it has 1 trick
"npm run test token.spec" will run "gulp token.spec"

registerJasmineTasks register many tasks so can't register all of them to npm scripts.
so I've using "test" for wildcard gulp task.

**Pull request merged**